tests/integration/jobs/jobs.spec.ts

```diff
@@ -418,7 +419,7 @@ describe('job', function () {
 
         expect(response.status).toBe(httpStatusCodes.OK);
         expect(jobsFindMock).toHaveBeenCalledTimes(1);
-        expect(jobsFindMock).toHaveBeenCalledWith({ relations: ['tasks'], where: {} });
+        expect(jobsFindMock).toHaveBeenCalledWith({ relations: ['tasks'], where: {}, select: expect.arrayContaining([expect.any(String)]) });
```

Comment:
[expect.not.arrayContaining(array)](https://jestjs.io/docs/expect#expectnotarraycontainingarray)
Let's try this
expect.not.arrayContaining('parameters')

Reply:
Done